CHANGELOG.md

```diff
@@ -3,6 +3,20 @@
 
 ## Unreleased
 
+### Breaking Change
+
+- Remove the `createSubaccount` method from `Account` and `MuxedAccount` ([#487](https://github.com/stellar/js-stellar-base/pull/487)). You should prefer to do this manually:
```

Comment:
When I suggested breaking it out I had imagined this part, a functional change, would stay in the other PR, but either way this was easy to review. 👏🏻

Reply:
In retrospect that would've been cleaner, but thanks for the review! :heavy_check_mark: